src/components/MyAccount/ItemsTab.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Icon, Link, Table } from "@nypl/design-system-react-components"
+import styles from "../../../styles/components/MyAccount.module.scss"
+
+const ItemsTab = ({
+  headers,
+  data,
+  userAction,
+}: {
+  headers: string[]
+  data: any[]
+  userAction: "requested" | "checked out"
+}) => {
+  return (
+    <>
+      {data?.length === 0 && (
+        <Box className={styles.notification}>
+          <span>
+            You currently do not have any research items {userAction}.
```

Comment:
I'm confirming with Apoorva to see if this is what we want to say here.

Reply:
She says:
> Good point! Changing the language to just 'items requested'

Reply:
But should the checkouts tab still specifically say "research items checked out" ?

Reply:
You can remove the word "research".